### PR TITLE
add macOS instruction using homebrew

### PR DESCRIPTION
**Pull request by thiolliere**
It tooks me some times to figure out that I had to pass --with-python3 argument

I think those instruction might help

### CODE REVIEW

**Comment by lazka** [comment]
Thanks. Will that also install gtk+3 and pycairo? It would be nice if the instructions would lead to about the same set of installed software on all platforms.

**Comment by thiolliere** [comment]
the hello.py example works. So gtk+3 is installed, isn't it ?
and `import cairo` works. and the formula says:
https://github.com/Homebrew/homebrew-core/blob/43801bc55931538c39cf82ff542b5f5139906852/Formula/pygobject3.rb#L20-L21

**Comment by lazka** [comment]
> the hello.py example works. So gtk+3 is installed, isn't it ?

Yeah. But the question is, is it also the case for users following the instructions you propose?
Would a "brew install gtk+" be needed as well here?

> and the formula says: […]

ok, that looks like it takes care of pycairo at least.

File: getting_started.rst [hunk 1]
```diff
@@ -83,4 +83,8 @@ installation.
 |macosx-logo| macOS
 -------------------
 
-1) TODO: complicated, jhbuild
+1) Go to https://brew.sh/ and install homebrew
+2) Open a terminal
+3) Execute ``brew install pygobject3 --with-python3``
+4) Change the directory to where your ``hello.py`` script can be found (e.g. ``cd Desktop``)
+5) Run ``python2 hello.py``
```
Comment (virtuald) [inline getting_started.rst]: Hm? why python2?

Comment (thiolliere) [inline getting_started.rst]: on linux distributions the documentation install for python2 and python3 and the run command runs with python2:
https://github.com/pygobject/pygobject-docs/blob/master/getting_started.rst#ubuntu-logo-ubuntu--debian-logo-debian

Comment (virtuald) [inline getting_started.rst]: Yeah, but you specified `--with-python3`? Does it install for both python 2 and 3, or just one? (you should clarify this)

**Comment by virtuald** [comment]
In the past, using Gtk+ from homebrew required using X11 interface to show the GUI. Is that still the case? If so, that should be noted.

File: getting_started.rst (hunk 1 above)
Comment (virtuald) [inline getting_started.rst]: Ok, then that should be added to the docs so someone else doesn't get confused too. :)

Comment (virtuald) [inline getting_started.rst]: Oh, I see. I missed that. That's a bit weird, but I guess you're right, that's consistent with the rest of the docs.

**Comment by lazka** [comment]
Looks like it uses the quartz backend: https://github.com/Homebrew/homebrew-core/blob/master/Formula/gtk+3.rb

**Comment by thiolliere** [comment]
I thought I didn't manually installed gtk+3 but now I'm not sure.
if I uninstall gtk+3 and pygobject3 and then install pygobject3 then it lacks gtk+3 so...

**Comment by lazka** [comment]
Thanks!

**Pull request merged**